www/collision.html

```diff
@@ -5,8 +5,8 @@
 </head>
 <body>
   <script src="https://d3js.org/d3.v3.min.js"></script>
-  
-  <script src="./webgazer.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/localforage/1.8.1/localforage.js"></script>
+  <script src="../dist/webgazer.var.js"></script>
```

Comment:
Can you explain what the difference is between var and common etc...

Reply:
In the webpack config we have src/index.js as entry file which exports webgazer object by default. 
this is the output config -
filename: `webgazer.${options.target}.js`,
      library: 'webgazer',
      libraryTarget: options.target,
      libraryExport: 'default',
      path: path.resolve(__dirname, 'dist'),

when library target is var then we get output file as 
var wegazer = webgazer_object_export_from_src_index;
when library target is commonjs2 then output file is like
module.export = webgazer_object_export_from_src_index;
var is useful when someone wants to include output script throught script tag in the browser.
commonjs2 is useful when someone wants to include the project into react or any other modern framework.

to make it more as a library we have to filter out the core functionality and put it out as a npm/yarn package.

Reply:
Okay, can we also generate non-minified code snippets for the Github. The minified code snippits rapidly pollute and bloat the size of the Github repo. (It looks like it will be disabled if the mode is not production).

Reply:
I'm confused about this - I'm trying to get a slightly modified version of this code running but getting

> http://localhost:3000/dist/webgazer.var.js net::ERR_ABORTED

When I run locally, here's my branch

https://github.com/RobinReborn/WebGazer

Reply:
This was fixed by #150 . Sorry, should have realized that the symlink wasn't setup properly. It also doesn't quite work on Windows so we are looking for better long term solution.